synfig-studio/src/gui/docks/dock_info.cpp

```diff
@@ -177,6 +177,12 @@ studio::Dock_Info::Dock_Info()
 	render_progress.set_text(strprintf("%.1f%%", 0.0));
 	render_progress.set_fraction(0.0);
 
+	Gtk::Button *stop_button = manage(new Gtk::Button());
+	stop_button->set_label("Stop");
+	stop_button->signal_clicked().connect(sigc::mem_fun(*this, &studio::Dock_Info::stop_renderer));
```

Comment:
Particularly I prefer the gtk naming convention for callbacks: on_stop_button_clicked. But just my taste.

Reply:
Done, better to keep things consistent